exercises/robot-simulator/example.h

```diff
@@ -0,0 +1,27 @@
+#if !defined(ROBOT_SIMULATOR_H)
+#define ROBOT_SIMULATOR_H
+#include <string>
+#include <utility>
+
+namespace robot_simulator {
+
+enum class Bearing {NORTH, WEST, SOUTH, EAST};
+
+class Robot 
+{
+public:
+    Robot();
+    Robot(std::pair<int, int>, Bearing);
+    void turn_left(); 
+    void turn_right();
+    void advance();
+    void execute_sequence(std::string);
```

Comment:
Use a const string reference here.

Reply:
OK, changed this. No need to copy the string, really...